apidef/schema.go

```diff
@@ -1,857 +1,1123 @@
 package apidef
 
 const Schema = `{
-    "type": ["object", "null"],
-    "$schema": "http://json-schema.org/draft-04/schema",
-    "id": "http://jsonschema.net",
-    "additionalProperties": false,
-    "properties": {
-        "is_site": {
-            "type": "boolean"
-        },
-        "uptime_tests": {
-            "type": ["object", "null"]
-        },
-        "expire_analytics_after": {
-            "type": "number"
-        },
-        "id": {
-            "type": "string"
-        },
-        "org_id": {
-            "type": "string"
-        },
-        "api_id": {
-            "type": "string"
-        },
-		"expiration": {
-            "type": "string"
-        },
-        "tags_disabled": {
-            "type": "boolean"
-        },
-        "enable_ip_whitelisting": {
-            "type": "boolean"
-        },
-        "enable_ip_blacklisting": {
-            "type": "boolean"
-        },
-        "enable_context_vars": {
-            "type": "boolean"
-        },
-        "strip_auth_data": {
+  "type": [
```

Comment:
any idea why this has so many changes?

Reply:
most likely because we haven't bothered with formatting it in a long time